webpack.config.js

```diff
@@ -1,19 +1,42 @@
-const path = require("path");
+var path = require('path');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
 module.exports = {
-    entry: ["./index.js"],
+    entry: './src/index.js',
     output: {
-        path: path.resolve(__dirname, "dist"),
-        filename: "js/[name].js"
+        path: path.resolve('dist'),
+        filename: 'bundle.js'
+    },
+    resolve: {
+        extensions: ['.js', '.jsx']
     },
     module: {
-        rules: [
```

Comment:
actually, you should use the `rules` option instead of loaders. That is more in keeping with the current api. Read the webpack documentation